compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/NullObservabiltySymbolCollector.java

```diff
@@ -0,0 +1,27 @@
+package org.wso2.ballerinalang.compiler.semantics.analyzer;
+
+import org.wso2.ballerinalang.compiler.spi.ObservabilitySymbolCollector;
+import org.wso2.ballerinalang.compiler.tree.BLangPackage;
+import org.wso2.ballerinalang.compiler.util.CompilerContext;
+
+import java.nio.file.Path;
+
+/**
+ * Null Object for ObserverbilitySymbolCollector interface.
+ */
```

Comment:
As a practice, we add `@since` annotation. Please add that.

Reply:
Added in the latest commit